modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2AnalyticsCloudFormSent.testcase

```diff
@@ -45,9 +45,69 @@ definition {
 	@description = "Automation ID: LPS-150680 | Test Summary: Verify if the same user submits the setup page in two different tabs or diferrent user submits the setup at the same project displays a message to the user that the form was sent"
 	@priority = "3"
 	test MessageConfirmingFormHasAlreadyBeenSent {
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account Analytics Cloud");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and join a project and Going through onboarding") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			CustomerPortal.enterOnProject(
+				status = "Expired",
+				title = "Test Account Analytics Cloud");
+
+			CustomerPortal.fullOnboardingFlow();
+		}
+
+		task ("Go to product Activation then open a new tab") {
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "Analytics Cloud");
+
+			Open.openInTheNewTab(locator1 = "http://localhost:8080");
+		}
+
+		task ("Go to CP 2.0 then go to Project and the product activation page") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		// TODO LPS-150680 MessageConfirmingFormHasAlreadyBeenSent pending implementation
+			CustomerPortal.enterOnProject(
+				status = "Expired",
+				title = "Test Account Analytics Cloud");
+
+			CP2NavigationMenu.goToProductActivation();
+
+			CP2NavigationMenu.clickOnProductButton(product = "Analytics Cloud");
+		}
```

Comment:
@styller1 can we do these steps by only using the URL of the Analytics Cloud product?

something like 
Open.openInTheNewTab(locator1 = "http://localhost:8080/ and add the rest of the URL");

Reply:
Yes, actually, I had initially thought of this, but it became very unstable and sometimes broke the test, because a route bug happens when we go from a control panel function to a project page. This way it became more stable because it avoids the route bug.

Reply:
ci:forward